cpp/data/sql_updates/ub_tools.22

```diff
@@ -0,0 +1 @@
+ALTER TABLE metadata_presence_tracer RENAME COLUMN metadata_field_name TO marc_field_tag
```

Comment:
length should be changed to 3 here as well

Reply:
Done!

Reply:
Multiple Problems:
- Please use MODIFY COLUMN syntax so it works on Ubuntu + CentOS
- Don't forget NOT NULL in ub_tools.22
- Why did you add record_type in this pull request? (it is used nowhere)
- If you add a new column, please also use the AFTER keyword to add it at the right place (else diffs will get more and more confusing)

Reply:
And please use semicolon after each statement!

Reply:
Done.